config/config.exs

```diff
@@ -1,4 +1,2 @@
-use Mix.Config
```

Comment:
If the config file is empty, should we just remove it?

Reply:
Done